pkgs/os-specific/linux/nvidia-x11/default.nix

```
@@ -71,6 +71,16 @@ rec {
     sha256_64bit = "sha256-Xagqf4x254Hn1/C+e3mNtNNE8mvU+s+avPPHHHH+dkA=";
     settingsSha256 = "sha256-ryUSiI8PsY3knkJLg0k1EmyYW5OWkhuZma/hmXNuojw=";
     persistencedSha256 = "sha256-/2h90Gq9NQd9Q+9eLVE6vrxXmINXxlLcSNOHxKToOEE=";
+
+    prePatch = "pushd kernel";
+    postPatch = "popd";
+
+    patches = [
+      # source: https://gist.github.com/joanbm/963906fc6772d8955faf1b9cc46c6b04
+      ./nvidia-470xx-fix-linux-6.2.patch
```

I think he meant it like:
```suggestion
      (fetchpatch {
        url = "https://gist.githubusercontent.com/joanbm/963906fc6772d8955faf1b9cc46c6b04/raw/0f99aa10d47b524aa0e6e3845664deac3a1ad9d9/nvidia-470xx-fix-linux-6.2.patch";
        hash = "sha256-***************";
       })
```

The comment would be in case of it being sourced from a PR.

---

Yes, the comment is to reference the URL that is source, not the patch itself.

---

Are you sure this should be applies to unofficial patches? Because a similar patch was merged in #206200 using the method I am currently using.

---

Ok, will check.

---

> Are you sure this should be applies to unofficial patches?

I am not aware that is a problem.

> Because a similar patch was merged in #206200 using the method I am currently using.

`fetchpatch` is cleaner, references patch source directly, it is usually the preferred way.

---

ok, will push an update in a minute